src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyTransport.php

```diff
@@ -115,8 +117,8 @@ protected function doSend(MessageInterface $message): SentMessage
 
     public function supports(MessageInterface $message): bool
     {
-        return $message instanceof PushMessage &&
-            (null === $message->getOptions() || $message->getOptions() instanceof NtfyOptions);
+        return $message instanceof PushMessage
+            && (null === $message->getOptions() || $message->getOptions() instanceof NtfyOptions);
```

Comment:
@keradus is there a PHP-CS-Fixer rule for this?

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/src/Fixer/Operator/OperatorLinebreakFixer.php ?